lib/corsica.ex

```diff
@@ -81,7 +80,7 @@ defmodule Corsica do
 
   For example:
 
-      # Matches everything
+      # Matches everything. Very insecure. Never use this in production.
```

Comment:
Let's not tell people what to do. We can mention that the use of `*` is *discouraged* in the paragraphs below.